Upgraded Alice to version 3

angelov opened the pull request:
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no?
| BC breaks?      | yes
| Related tickets | #99
| License         | MIT

I made some changes in order to make the library work with Alice 3. I'm not sure if there's a better way to achieve this, so I would like to hear what you'd say.

The persistence layer is removed from Alice 3, so an additional library is required. The simplest way to bring the Doctrine support back was to require two additional bundles:
* Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle
* Fidry\AliceDataFixtures\Bridge\Symfony\FidryAliceDataFixturesBundle

(The second bundle doesn't have a stable release yet, so if we go with this approach we'd have to wait for a bit.)

Upgrading to Alice 3 will probably break some fixtures in existing projects.

Because of this breaks, the major version of this library should be bumped.

PS. This will also require to change the minimal required PHP version to 7.1

lchrusciel commented on composer.json:
```diff
@@ -58,5 +59,6 @@
         }
-    }
+    },
+    "minimum-stability": "beta"
```
RC

lchrusciel commented on composer.json:
```diff
@@ -30,13 +30,14 @@
         "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/framework-bundle": "^3.2",
+        "theofidry/alice-data-fixtures": "^1.0@RC"
```
When will it be published?

angelov replied:
I don't know, the latest RC release is from 17 days ago.

lchrusciel replied:
Three days ago Theo released a stable version. Are you willing to update this PR, @angelov?

lchrusciel commented:
But I will wait a little bit with merging it. It would be nice to have Théo bundle stabalized. I have also considered decoupling of API test case and nelmio. We could just suggest its installation in composer and could be added to PHPUnit with the trait. It would reduce complication in the future. But I'm not able to do it before second part of December :(

angelov commented:
I understand. I'll see if I can do something to help in the meanwhile.

angelov commented:
@lchrusciel I changed `composer.json` to require `theofidry/alice-data-fixtures` v1.0. I also added some additional info to the readme file, please check it out.

lchrusciel commented:
Thanks Dejan! :)

lchrusciel commented:
Just taged version with this support :)
